modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/context/BaseCommerceContextHttp.java

```diff
@@ -204,27 +204,18 @@ public CommerceOrder getCommerceOrder() {
 				return null;
 			}
 
-			HttpServletRequest originalHttpServletRequest =
-				_portal.getOriginalServletRequest(_httpServletRequest);
+			_commerceOrder = _commerceOrderHttpHelper.getCurrentCommerceOrder(
+				_httpServletRequest);
 
-			HttpSession httpSession = originalHttpServletRequest.getSession();
-
-			long groupId = commerceChannel.getGroupId();
+			if (_commerceOrder != null) {
+				HttpServletRequest originalHttpServletRequest =
+					_portal.getOriginalServletRequest(_httpServletRequest);
 
-			String uuid = (String)httpSession.getAttribute(
```

Comment:
this change would have been easier to read if you put this in the following commit

Reply:
The fix that occurred here led to the changes made in the following commit, but I can squash them if it is easier to read

Reply:
no no worries it's just a matter of code review